soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3276,12 +3276,16 @@ CustomMessage Randomizer::GetGoronMessage(u16 index) {
     return messageEntry;
 }
 
+std::string CUSTOM_ITEM_ICON(uint8_t customIconIndex, std::string msg) {
+    return char(customIconIndex) + msg;
+}
```

Comment:
instead of this it might be good to make something that does everything `GIMESSAGE` does but also adds the custom item index